### ISSUE TITLE
[DOC] Data Prepper OpenSearch sink add aws serverless options field

### ISSUE THREAD

**Issue by asuresh8** [opensearch-project/documentation-website#5321]
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [X] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 
Adding a new serverless options field in the aws section of the opensearch sink. See below for example

```
  1 version: "2"
  2 log-pipeline:
  3   source:
  4     http:
  5       path: "/test/path"
  6   processor:
  7     - grok:
  8         match:
  9           log: [ "%{COMMONAPACHELOG}" ]
 10     - date:
 11         from_time_received: true
 12         destination: "@timestamp"
 13   sink:
 14     - opensearch:
 15         hosts: [ "https://cw4c***************.us-east-1.aoss.amazonaws.com" ]
 16         index: "test_index"
 17         aws:
 18           sts_role_arn: "arn:aws:iam::*************:role/****************Role"
 19           region: "us-east-1"
 20           serverless: true
 21           serverless_options:
 22             network_policy_name: "new-adi-test-collection"
 23             collection_name: "adi-test-collection"
 24             vpce_id: "vpce-**********"
```

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

See https://github.com/opensearch-project/data-prepper/pull/3510 for the pull request with the code.

**Comment by hdhalter** [comment]
@Naarcha-AWS - Does this [PR](https://github.com/opensearch-project/documentation-website/pull/5663) close this doc issue? And I assume this is for version 2.6?

**Comment by Naarcha-AWS** [comment]
Yes, we can close this one. It was addressed by #5663
